rsconnect/main.py

```diff
@@ -220,6 +220,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def cloud_content_args(func):
+    @click.option(
+        "--project-application-id",
+        "-P",
+        envvar="LUCID_APPLICATION_ID",
+        help="The application of the project to to be associated with when creating an output.",
+    )
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        return func(*args, **kwargs)
+
+    return wrapper
```

Comment:
Would it be possible to reuse the existing app_id in content_args? Or are there technical limitations behind us adding a specific lucid app id?
content_args:
https://github.com/rstudio/rsconnect-python/blob/a2d544522c78e6e0a28719300a08a4826baea046/rsconnect/main.py#L190-L204

Reply:
This is a different id from the existing app_id – see [Sean's comment](https://github.com/rstudio/rsconnect-python/pull/278#discussion_r963639303). But I removed it as a CLI option (keeping the environment variable) because it's not really useful in the user-facing interface.